### ISSUE TITLE
Future enhancements to the documentation landing page

### ISSUE THREAD

**Issue by rmglennon** [mapzen/documentation#96]
Let's consider doing some enhancements to the https://mapzen.com/documentation/ landing page when we get a chance. 

A few suggestions:
- The blocks are named based on products, but not everyone landing on the page will be familiar with them.
- There is a slight order to put some blocks at the top, but the blocks look the same without grouping or hierarchy overall.
- Could use a "get started" block to introduce someone new to products. See #75 for more on tutorials.
- Add some images to the page. I had this suggestion from a user: when you hover over a block, instead of changing it to a green background, have an image that represents what it does. 

cc @souperneon @migurski -- just putting this here for future thought, esp when there are any product reorgs or launches.

**Comment by souperneon** [comment]
these are all good ideas @rmglennon!

**Comment by migurski** [comment]
I am moving the documentation index to this repository; will be complete once #116 is done.

**Comment by rmglennon** [comment]
This page is now here: https://github.com/mapzen/mapzen-docs-generator/blob/master/src-index/index.md

**Comment by rmglennon** [comment]
Closing this...the page was updated long ago to be grouped by functionality and not product names, and we have added a get started section (adding a tutorials list is being tracked in another issue). The weird green hover state is gone, too. We could consider images in the future as part of a redesign.
